### ISSUE TITLE
Add wrapper for dbstmt.asNumber()

### ISSUE THREAD

**Issue by abmusse** [IBM/nodejs-idb-pconnector#54]
[idb-connector]() has support to return numeric data as a JS Number instead of a JS String when safe to do so.

This is done by calling [dbstmt.asNumber](https://github.com/IBM/nodejs-idb-connector/tree/master/docs#dbstmtasnumberflag)

We should a wrapper for this in [statement.js](https://github.com/IBM/nodejs-idb-pconnector/blob/master/lib/statement.js)

**Comment by larsandersson-ifpi** [comment]
has enableNumericTypeConversion been replaced with someting else?
i can not use it on the statement.

**Comment by kadler** [comment]
Nope, it's right [here](https://github.com/IBM/nodejs-idb-pconnector/blob/master/lib/statement.js#L495), but you probably can't use it because we haven't released a version with that change included yet.
